src/cpu.rs

```diff
@@ -134,7 +134,7 @@ impl Cpu {
             ),
         };
 
-        (pc, instruction.cycles())
+        (pc, instruction.cycles(false))
```

Comment:
Hmm, was this based on an older version of the code? In my last update with the macros, it should already be taking care of the conditional cycles work, so you shouldn't need a boolean value for cycles()